internal/vendors/syncer.go

```diff
@@ -53,7 +53,8 @@ func (s *Syncer) Sync(ctx context.Context) (err error) {
 	for _, firmware := range s.firmwares {
 		if err = s.syncFirmware(ctx, firmware); err != nil {
 			msg := fmt.Sprintf("failed to sync firmware %s", firmware.Filename)
-			return errors.Wrap(err, msg)
+			// Log error without returning, to sync other firmwares
+			s.logger.WithError(err).Error(msg)
```

Comment:
To make the logs easier to parse and search, we should be putting variables (such as `firmware.Filename`) into their own fields, like this:
```go
s.logger.WithError(err).WithField("firmware", firmware.Filename).Error("Failed to sync firmware")
```

Reply:
ok, it's better, thanks!